package.json

```diff
@@ -37,6 +37,7 @@
     "eslint": "5.0.1",
     "eslint-loader": "2.0.0",
     "eslint-plugin-vue": "4.0.0",
+    "file-loader": "^4.1.0",
```

Comment:
Remove the ^
On production we'll make builds every time we deploy. That way production will have the latest patched versions while locally you keep the old stuff. Production and local will be offset and some bugs will creep out (especially with JS amateurs calling themselves library maintainers). You can run this in the terminal

`npm config set save-exact true`

That will never set ^ by default. It will save you a bunch of stress in the future ;)
The only argument against this is that you may not automatically get security patches. With the new npm and Github warnings about vulnerabilities I'm not too worried about missing them. We'll know and run fixes when needed.